externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -1282,6 +1282,26 @@ abstract class FlinkOperationSuite extends HiveJDBCTestHelper with WithFlinkTest
     assert(exception.getMessage === "Futures timed out after [60000 milliseconds]")
   }
 
+  test("test result fetch timeout but already fetched some data") {
+    withSessionConf()(Map(
+      ENGINE_FLINK_MAX_ROWS.key -> "10",
+      ENGINE_FLINK_FETCH_TIMEOUT.key -> "PT5S"))(Map.empty) {
+      withJdbcStatement() { statement =>
+        statement.execute("create table tbl_src_a (a bigint) with (" +
```

Comment:
Why do we need 2 tables in the test?

Reply:
one is bounded data ,it  only have one rows, the other is  unbounded data, the union query is  just like we query from kafka,but it have only one rows.